Rollbar/RollbarLogger.cs

```diff
@@ -488,7 +468,23 @@ void IDisposable.Dispose()
                 timeoutAt = DateTime.Now.Add(timeout.Value);
             }
             // we are taking here a fire-and-forget approach:
-            Task.Factory.StartNew(() => Send(body, level, custom, timeoutAt, signal));
+            Task task = Task.Factory.StartNew(() => Send(body, level, custom, timeoutAt, signal));
+            bool success = false;
+            while(!success)
+            {
+                success = this._pendingTasks.TryAdd(task, task);
```

Comment:
Why `task, task`? Is there some reason to set the value to the task or is it because there isn't anything else useful, i.e. you'd use a ConcurrentSet if it existed but it doesn't?

Reply:
yes, ConcurrentSet does not exist yet, so i am using ConcurrentDictionary for now (probably better than using non-concurrent set with custom locking)